bananalyzer/runner/runner.py

```diff
@@ -130,7 +130,7 @@ def run_tests(
         args = (
             test_file_names
             + ["-s"] * pytest_args.s
-            + (["-q"] if pytest_args.q else ["-vvv"])
+            + (["-vvv"] if pytest_args.v else ["-v"])
```

Comment:
-v is enough to not show multiple error lines?

Reply:
yup, tested